src/pages/home/sidebar/FloatingActionButtonAndPopover.tsx

```diff
@@ -454,6 +455,85 @@ function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu, isT
     const canModifyTask = canModifyTaskUtils(viewTourTaskReport, currentUserPersonalDetails.accountID);
     const canActionTask = canActionTaskUtils(viewTourTaskReport, currentUserPersonalDetails.accountID);
 
+    const menuItems = [
+        ...expenseMenuItems,
+        {
+            icon: Expensicons.ChatBubble,
+            text: translate('sidebarScreen.fabNewChat'),
+            onSelected: () =>
+                interceptAnonymousUser(() => {
+                    InteractionManager.runAfterInteractions(() => {
```

Comment:
This is now nested `runAfterInteractions`. Do you think this could cause some issues